packages/vue/test/base/tests/e2e/specs/tabs.cy.js

```diff
@@ -1,4 +1,43 @@
 describe('Tabs', () => {
+  /**
+   * Verifies that tabs with similar route prefixes (e.g., /home, /home2, /home3)
+   * correctly select the matching tab instead of the first prefix match.
```

Comment:
```suggestion
   * correctly select the matching tab instead of the first prefix match.
   * 
   * @see https://github.com/ionic-team/ionic-framework/issues/30448
```